design-docs/CLI-Subscriptions.md

```diff
@@ -175,6 +175,49 @@ In the case of invalid options, print an error message (stderr) and set the exit
 code to 2. In the case of an API error response, print the response JSON and 
 set code to 1.
 
+## Request
+
+### Interface
+
+```
+planet subscriptions request [OPTIONS]
+
+Generate a subscriptions request.
+
+This command provides support for building the subscription request JSON used to create or 
+update a subscription. It outputs the subscription request.
+
+Options:
+  --name TEXT                     Subscription name. Does not need to be unique.
+                                  [required]
+  --asset-types TEXT              One or more comma-separated asset types [required]
+  --item-types TEXT               One or more comma-separated item-types [required]
+  --geometry JSON                 GeoJSON of area to deliver subscription to. Use 
+                                  --clip only deliver pixels that match this area.
+                                  Can be a json string, filename, or '-' for stdin.
+  --clip                          Use the geometry GeoJSON as the clip geometry for 
+                                  the subscription.
+  --start-time DATETIME           Start date and time to begin subscription.
+  --end-time DATETIME             Date and time to end the subscription.
+  --rrule TEXT                    iCalendar recurrance rule to specify recurrances. 
+  --filter JSON                   A search filter can be specified a json string, 
+                                  filename, or '-' for stdin. 
+  --tools JSON                    Toolchain JSON. Can be a json string,
+                                  filename, or '-' for stdin.
+  --cloudconfig JSON              Credentials for cloud storage provider to
```

Comment:
We called this `cloudconfig` in orders, so keeping it consistent here, but the name of the block in both orders and subscriptions is `delivery`, so we could potentially call it that too...